checker/jtreg/stubs/issue1496/Main.java

```diff
@@ -0,0 +1,9 @@
+/*
+ * @test
+ * @summary Test case for Issue 1496 https://github.com/typetools/checker-framework/issues/1496
+ * @compile -XDrawDiagnostics -processor org.checkerframework.checker.nullness.NullnessChecker -Astubs=ClassAnnotation.astub Main.java
+ * */
+
+package issue1496;
+
+public class Main extends Object {}
```

Comment:
I would remove the `extends Object`, as it doesn't serve a purpose for this test.